dash_table/Format.py

```diff
@@ -1,6 +1,4 @@
 import collections
-import inspect
-import sys
```

Comment:
dunno why, but locally `npm run lint` asked me to make the changes in this PR, even though on CI it passes 🤔

Reply:
Running the same versions? I've found that the default rules are updated frequently..

Reply:
oh actually `npm run lint` excluded a whole bunch of rules that we really don't need to exclude (as long as we're excluding the auto-generated files, which we are). I'll un-exclude them. Probably worth taking a look at this in other repos as well... it makes sense to have looser linting in tests but some of these rules are useful - like unused imports, can improve perf, only files setting up namespaces have any reason to include them.

Reply:
stricter linting -> https://github.com/plotly/dash-table/pull/412/commits/8f01c257b200d97bee49eb2e344ab46438d4fba9